opm/input/eclipse/Schedule/Group/Group.hpp

```diff
@@ -195,6 +195,9 @@ struct GroupProductionProperties {
     std::string name;
     ProductionCMode cmode = ProductionCMode::NONE;
     ExceedAction exceed_action = ExceedAction::NONE;
+    ExceedAction water_exceed_action = ExceedAction::NONE;
+    ExceedAction gas_exceed_action = ExceedAction::NONE;
+    ExceedAction liquid_exceed_action = ExceedAction::NONE;
```

Comment:
Could you maybe collect these in a structure instead?  For instance something like
```C++
struct GroupLimitAction
{
    ExceedAction phaseRate{ExceedAction::NONE};
    ExceedAction water{ExceedAction::NONE};
    ExceedAction gas{ExceedAction::NONE};
    ExceedAction liquid{ExceedAction::NONE};
};
```
That way it's easier to refer to all of these actions as a group and we don't need to add more parameters to every function that needs them.